src/Knp/FriendlyContexts/Context/CommandContext.php

```diff
@@ -141,6 +141,18 @@ public function commandOutputShouldBeLike(PyStringNode $string)
     }
 
     /**
+     * @Then I should see :string in the command output
```

Comment:
`\PHPUnit_Framework_Assert::assertContains` calls `\PHPUnit_Framework_Constraint_StringContains::matches`, which executes an `strpos` (or `stripos` if necessary), which is exactly the same.